packages/build/src/local/trigger-release-draft.ts

```diff
@@ -0,0 +1,98 @@
+import assert from 'assert';
+import semver from 'semver';
+import { choose, confirm, spawnSync as spawnSyncFn } from '../helpers';
+import { getLatestDraftOrReleaseTagFromLog, TagDetails } from './get-latest-tag';
+import { getRepositoryStatus } from './repository-status';
+
+type BumpType = 'draft' | 'patch' | 'minor' | 'major';
+
+export async function triggerReleaseDraft(
+  repositoryRoot: string,
+  spawnSync: typeof spawnSyncFn = spawnSyncFn
+): Promise<void> {
+  console.info('Triggering process to create a new release draft...');
+
+  verifyGitStatus(repositoryRoot);
+
+  const latestDraftOrReleaseTag = getLatestDraftOrReleaseTagFromLog(repositoryRoot);
+  if (!latestDraftOrReleaseTag) {
+    throw new Error('Could not find a previous draft or release tag.');
+  }
+  console.info(`-> Most recent tag: v${latestDraftOrReleaseTag.tag.semverName} on commit ${latestDraftOrReleaseTag.commit}`);
+
+  let bumpType: BumpType = 'draft';
+  if (latestDraftOrReleaseTag.tag.draftVersion === undefined) {
+    bumpType = await choose('>  Select the type of increment for the new version', [
+      'patch', 'minor', 'major'
+    ], '... enter your choice:') as BumpType;
+  }
+
+  const nextTagName = computeNextTagName(latestDraftOrReleaseTag.tag, bumpType);
+  console.info('-> New draft tag is:');
+  console.info(`       ${nextTagName}`);
+
+  const confirmed = await confirm('!! Is this correct and should the draft process continue?');
+  if (!confirmed) {
+    throw new Error('User aborted.');
+  }
+
+  console.info('... creating and pushing tag ...');
+  spawnSync('git', ['tag', nextTagName], {
+    cwd: repositoryRoot,
+    encoding: 'utf-8'
+  });
+  spawnSync('git', ['push', '--tags'], {
```

Comment:
I would maybe recommend only pushing the tag that we created here, instead of any tags the user might have lying around as well

Reply:
I currently fail if the user has unpushed tags - so here it would not push anything additional. But good point, might change that to only push that specific tag.